src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/pipelines/PipelinesPhylogenomicsPage.java

```diff
@@ -20,8 +20,8 @@
  * Phylogenomics workflow launch page.
  *
  */
-public class PipelinesPhylogenomicsPage extends AbstractPage {
-	
+public class PipelinesPhylogenomicsPage extends BasicPipelinePage {
```

Comment:
Did a wee refactor here to create a `BasicPipelinePage` and have `PipelinesPhylogenomicsPage` extend it.  That way we could potentially test other specialized pages.